### PR TITLE
doc: add description for containerized-test to development-guide

### PR DESCRIPTION
**Pull request by nixpanic**
As suggested by @wilmardo in #797 add a description of how to run the tests locally.

### CODE REVIEW

File: docs/development-guide.md [hunk 1]
```diff
@@ -44,6 +44,26 @@ To build ceph-csi in a container:
 
 The built binary will be present under `_output/` directory.
 
+### Running Ceph-CSI tests
+
+Once the changes to the sources compile, it is good practise to run the tests
+that validate the style and other basics of the source code. Execute the unit
+tests (in the `*_test.go` files) and check the formatting of YAML files and
```
Comment (Madhu-1) [inline docs/development-guide.md:51]: it also validates the .md files

File: docs/development-guide.md [hunk 2]
```diff
@@ -44,6 +44,26 @@ To build ceph-csi in a container:
...
+shell scripts:
+
+`$ make containerized-test`
```
Comment (Madhu-1) [inline docs/development-guide.md:54]: this is more specific containerized tests. need to update the title or put a subtitle as `running containerized test`

File: docs/development-guide.md (hunk 1 above)
Comment (nixpanic) [reply docs/development-guide.md]: yes, and more. But I added "MarkDown documents" too now.

File: docs/development-guide.md (hunk 2 above)
Comment (nixpanic) [reply docs/development-guide.md:54]: Yeah, that makes sense. I was not going to explain how to run the tests outside of a container as it requires installing quite some dependencies. Title has been updated.

**Pull request merged**